lib/util.js

```diff
@@ -517,7 +517,7 @@ function formatCollectionIterator(ctx, value, recurseTimes, visibleKeys, keys) {
   var nextRecurseTimes = recurseTimes === null ? null : recurseTimes - 1;
   var vals = mirror.preview();
   var output = [];
-  for (let o of vals) {
+  for (const o of vals) {
```

Comment:
Probably unrelated to strong mode at least, can't even get a strong mode REPL up:

````console
$ node --use_strong
node.js:14
    var EventEmitter = NativeModule.require('events');
    ^^^

SyntaxError: In strong mode, 'var' is deprecated, use 'let' or 'const' instead
````